dev/PushNotifications/PushNotificationChannel.cpp

```diff
@@ -35,12 +36,16 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         try
         {
             m_channel.Close();
+
+            PushNotificationTelemetry::ChannelClosedByApi(S_OK);
         }
         catch (...)
         {
             auto channelCloseException = hresult_error(to_hresult());
             if (channelCloseException.code() != HRESULT_FROM_WIN32(ERROR_NOT_FOUND))
```

Comment:
What if the error is ERROR_NOT_FOUND? Don we want to log this too?

Reply:
Yes we do. Thanks for cathing this.